overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/ae2/misc.groovy

```diff
@@ -117,7 +117,7 @@ var applyProcessorChange = { OreDictIngredient lens, OreDictIngredient plate, It
 		.top(circuit)
 		.bottom(item('appliedenergistics2:material:20')) // Silicon
 		.output(processor)
-		.inscribe().register()
```

Comment:
Should this line have been changed?

Reply:
Yes, the inscribe and press behaviour has been swapped so keeping it as inscribed means the parts aren't used up

Reply:
I see, thanks.